src/momlevel/trend.py

```diff
@@ -1,7 +1,7 @@
 """ trend.py - utilities for working with trends """
 
 import warnings
-
+import numpy as np
 import xarray as xr
 
 __all__ = [
```

Comment:
Please add `"seasonal_model",`to the `__all__` list.

Reply:
Added in the latest commit